docs/advanced/full-node.md

```diff
@@ -0,0 +1,63 @@
+- **Safety & Privacy:** Because a lightweight node cannot fully validate transactions and blocks, it must trust third parties to do so. This means that the third party will have access to information about your transactions, as well as any personal data you give to that third party. For instance, many lightweight wallets rely on a centralized server to process transactions. Even Simplified Payment Verification (SPV) wallets, which typically connect directly to full nodes in the network, still rely on those other full nodes to process and broadcast their transactions. 
+
+## Requirements:
+
+Below are the basic requirements for running a full node:
+
+- \>15 GB disk space (as of September 2019, increases over time)
+- \>2 GB of RAM
+- Stable Operating System (OS) &mdash;Windows 7/8.x/10 (server preferred), macOS, Linux
+- Reliable internet connection
+
+If running a node on your own hardware (i.e. not on remote servers), you will also need:
+
+- Access to the settings of your router (to allow outbound connections)
+- A safe place to leave your device running
+- If you need a silent node (hard disk drives can be loud), a SSD or USB drive
+
+## Guides
+
+Below are some guides from the community that walk you step-by-step through setting up a full node on various hardware and OSes. For generic high-level instructions, see the [Steps](#steps) section below. 
+
+- [Running a Decred Raspberry Pi Node](https://medium.com/@_Checkmatey_/running-a-decred-raspberry-pi-node-ac605b70c652): A more beginner friedly guide that uses a VNC server (remote desktop). This guide is intented to be a gentle introduction to using SSH on the command line. 
+- [My experience setting up a Decred full-node](https://medium.com/crypto-rocket-blog/my-experience-setting-up-a-decred-full-node-8a9bbf55bc30): Another beginner-friendly guide that walks through setting up a full node on a Raspberry Pi using an Ubuntu terminal on a Windows PC.
+- [Installing dcrd](https://stakey.club/en/installing-dcrd/): A more advanced guide to installing a full node (Mac or Debian) that also covers manual installation (from binaries or source) and configuring dcrd to run on Tor for extra security.
+
+## Steps
+
+Below are the basic high-level steps to install a full node on any hardware and OS. These instructions use [dcrinstall](../wallets/cli/cli-installation.md), the automatic installer and upgrader software. In addition to installing `dcrd`, it will also install `dcrwallet` (which allows you to create wallets) and `dcrctl` (which allows you to control `dcrwallet` and `dcrd` from the command line). You can also just install `dcrd` on its own from [binaries or source files](https://github.com/decred/dcrd#installing-and-updating); a full node does not require a wallet unless you want to use your full node to validate and broadcast your own transactions.
+
+1. **Download and install dcrd**
```

Comment:
Numbered list should use `1)` on every line

Reply:
One day I will learn...